lib/output.js

```diff
@@ -1444,17 +1447,20 @@ function OutputStream(options) {
         }
     });
     DEFPRINT(AST_ObjectKeyVal, function(self, output){
-        var print_as_shorthand = self.shorthand &&
+        var value;
```

Comment:
`value` does not accurately reflect what it will hold. Please consider this line changing to:
```
var def, name;
```